tests/test_widerface_format.py

```diff
@@ -9,11 +9,16 @@
 from datumaro.components.environment import Environment
 from datumaro.components.extractor import DatasetItem
 from datumaro.components.media import Image
+from datumaro.components.operations import IntersectMerge
 from datumaro.plugins.widerface_format import WiderFaceConverter, WiderFaceImporter
 from datumaro.util.test_utils import IGNORE_ALL, TestDir, compare_datasets
 
 from .requirements import Requirements, mark_requirement
 
+MANGLING_DATASET_DIR = osp.join(
+    osp.dirname(__file__), "assets", "widerface_dataset", "mangling_dataset"
+)
```

Comment:
What is the difference between mangled datasets and example datasets? Do we need them all?

Please rename the directory from `mangling_...` to `mangled_...`